### DISCUSSION TITLE
스타일 전략

### DISCUSSION THREAD

**Discussion by codeisneverodd** [prgrms-fe-devcourse/FEDC2_TimeDivider_Root#6] [0. 개발]
styled-component 사용
emotion은 사용하지 않음

**Comment by dahye1013** [comment]
저희 사용할 색상을 상수로 지정해야할 것 같아요!
지금은 하드코딩 되어있어서 구분이 좀 어렵게 느껴지네요.
primary secondary tertiary 혹은 컬러팔레트에서 적절한 색상 조합을 선정해봐요.
